.github/scripts/ci-test.sh

```diff
@@ -13,7 +13,7 @@ fi
 
 ./examples/build.py
 
-ALL_PLANS=$(sed -n '/enum PlanSelector/,/}/p' src/util/options.rs | xargs | grep -o '{.*}' | grep -o '\w\+')
+ALL_PLANS=$(sed -n '/enum PlanSelector/,/}/p' src/util/options.rs | sed -e 's/\/\/.*//g' -e '/^$/d' -e 's/,//g' | xargs | grep -o '{.*}' | grep -o '\w\+')
```

Comment:
The `s` command of sed and vim also accept other separators in addition to `/`.  For example, it will be more readable if we write `s;//.*;;g` instead.

Reply:
I have made changes as suggested.